fixit/cli/add_new_rule.py

```diff
@@ -57,7 +57,7 @@ class {class_name}Rule(CstLintRule):
 
 
 def is_path_exists(path: str) -> Path:
-    """Check for valid path, if yes, return `Path` else raise `Error` """
+    """Check for valid path, if yes, return `Path` else raise `Error`"""
```

Comment:
These changes were added automatically with `tox -e autofix`

Reply:
Made https://github.com/Instagram/Fixit/pull/190 to handle this separately, I will need to rebase these changes once that is landed.